Ansible/todoapp.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=todoapp
+
+[Service]
+User=ec2-user
+WorkingDirectory=/opt/todoapp
+ExecStart=/home/ec2-user/.local/bin/poetry run flask run --host 0.0.0.0
```

Comment:
This isn't actually succeeding - if you run `sudo journalctl -u todoapp` to check the service logs, you'll see it errors because poetry isn't being found.

It looks like you've installed poetry as the root user, so we'll need to track down their version of poetry and ensure that we're using that, and we'll probably need to run this as root. That said, a _better_ approach would be to install poetry under the ec2-user, so that we're not using admin permissions unnecessarily

So the quickest solution is I believe:
* Update this file to have:
```
User=root
```
and
```
ExecStart=/root/.local/bin/poetry run flask run --host 0.0.0.0
```
(Note here where the poetry executable is installed, which is slightly different to where it would be for the ec2-user

But a better approach would be to run the `- name: install poetry` step without `become` so that it installs poetry for the `ec2-user`